### PR TITLE
added validation test

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

**Comment by trapacska** [comment]
as you can see the second commit had the working version of validation test, and by the last commit I pushed the fix for the issue which should lead to a green CI

File: _tests/integration/test_deploy_structure_check.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+format_version: 3
```
Comment (BirmacherAkos) [inline _tests/integration/test_deploy_structure_check.yml]: Could you update the format_version, please?

File: configs/paths.go [hunk 2]
```diff
@@ -34,9 +34,9 @@ const (
 	BitriseSourceDirEnvKey = "BITRISE_SOURCE_DIR"
 	// BitriseDeployDirEnvKey ...
 	BitriseDeployDirEnvKey = "BITRISE_DEPLOY_DIR"
-	// BitriseTestDeployDirEnvKey ...
+	// BitriseTestDeployDirEnvKey contains a path from Deploy to Bitrise.io step will read each test results (all subdirs / from all steps)
 	BitriseTestDeployDirEnvKey = "BITRISE_TEST_DEPLOY_DIR"
```
Comment (godrei) [inline configs/paths.go]: i would recommend: `The root directory of test reports`

File: configs/paths.go [hunk 3]
```diff
@@ -34,9 +34,9 @@ const (
...
-	// BitriseTestResultDirEnvKey ...
+	// BitriseTestResultDirEnvKey contains a path where test results from a single step can be copied over and will be deployed
```
Comment (godrei) [inline configs/paths.go]: `A unique subdirectory in BITRISE_TEST_DEPLOY_DIR for each step run, steps should place test reports and attachments into this directory`

File: _tests/integration/test_deploy_structure_check_test.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+)
+
+func Test_DeployDirStructure(t *testing.T) {
+	t.Log("check test deploy dir structure")
+	{
+		configPth := "test_deploy_structure_check.yml"
+
+		cmd := command.New(binPath(), "run", "test-deploy-dir-structure-check", "--config", configPth)
+		out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+		require.NoError(t, err, out)
```
Comment (lpusok) [inline _tests/integration/test_deploy_structure_check_test.go]: Please do not use testify in new tests

File: configs/paths.go [hunk 5]
```diff
@@ -34,9 +34,9 @@ const (
 	BitriseSourceDirEnvKey = "BITRISE_SOURCE_DIR"
 	// BitriseDeployDirEnvKey ...
 	BitriseDeployDirEnvKey = "BITRISE_DEPLOY_DIR"
-	// BitriseTestDeployDirEnvKey ...
+	// BitriseTestDeployDirEnvKey is the root directory of test reports
 	BitriseTestDeployDirEnvKey = "BITRISE_TEST_DEPLOY_DIR"
-	// BitriseTestResultDirEnvKey ...
+	// BitriseTestResultDirEnvKey is a unique subdirectory in BITRISE_TEST_DEPLOY_DIR for each step run, steps should place test reports and attachments into this directory
```
Comment (lpusok) [inline configs/paths.go]: Could call it BitrisePerStepTestResultDirEnvKey to make it more descriptive when referencing

**Pull request merged**